tests/index.test.js

```diff
@@ -0,0 +1,66 @@
+const request = require("supertest");
+const app = require("../app.js");
+const fs = require("fs");
+const { response } = require("../app.js");
+const validTestJson = { url: "https://www.test123.com/" };
+const ValidUrlJSON = {
+  url:
+    "https://www.google.com/search?q=dfsdfdsfdsfddHGFHHFGHFgfjhghjkhgjk&oq=dfsdfdsfdsfddsfdsfdsddsdfsdfdsfdsgfjhghjkhgjk&aqs=chrome..69i57.3965j0j15&sourceid=chrome&ie=UTF-8",
+};
+const InValidUrlJSON = { url: "httls://google.com" };
+
+const validUrlShorterId = 1615042437523;
+
+const validUrlResponse = {
+  success: true,
+  method: "POST",
+  message: {
+    originalURL:
+      "https://www.google.com/search?q=dfsdfdsfdsfddHGFHHFGHFgfjhghjkhgjk&oq=dfsdfdsfdsfddsfdsfdsddsdfsdfdsfdsgfjhghjkhgjk&aqs=chrome..69i57.3965j0j15&sourceid=chrome&ie=UTF-8",
+    shortURLid: 1615055119949,
+    creationDate: "6/3/2021",
+    redirectCount: 0,
+  },
+};
+const invalidUrlResponse = {
+  success: false,
+  method: "post",
+  message: "Invalid URL adress",
+};
```

Comment:
Great use of mocks. Next time I recommend you to use an object.
Try to not use `PascalCase` for variable names, only `camelCase`